src/coreComponents/physicsSolvers/fluidFlow/ReactiveCompositionalMultiphaseOBL.cpp

```diff
@@ -116,6 +116,9 @@ ReactiveCompositionalMultiphaseOBL::ReactiveCompositionalMultiphaseOBL( const st
     setInputFlag( InputFlags::OPTIONAL ).
     setDescription( "List of fluid phases" );
 
+
+  //addLogLevel( "logLevel >= 1 first nonlinear iteration", "Information on the source flux boundary conditions" );
```

Comment:
Is there missing logLevel docs for this class?

Reply:
still not adressed if `ReactiveCompositionalMultiphaseOBL` is in the scope of this PR.